administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -75,13 +75,18 @@ COM_JOOMLAUPDATE_UPDATE_LOG_FILE="File %s downloaded."
 COM_JOOMLAUPDATE_UPDATE_LOG_FINALISE="Finalising installation."
 COM_JOOMLAUPDATE_UPDATE_LOG_INSTALL="Starting installation of new version."
 COM_JOOMLAUPDATE_UPDATE_LOG_START="Update started by user %2$s (%1$s). Old version is %3$s."
+COM_JOOMLAUPDATE_UPDATE_LOG_UPLOAD="Uploading update file"
 COM_JOOMLAUPDATE_UPDATE_LOG_URL="Downloading update file from %s."
+COM_JOOMLAUPDATE_UPDATE_LOGGING_TEST_FAIL="Logging does not work. Make sure the log folder is writable, and try again. The logging test failed with message: %s"
 COM_JOOMLAUPDATE_UPDATING_HEAD="Update in progress"
 COM_JOOMLAUPDATE_UPDATING_INPROGRESS="Please wait; Joomla is being updated. This may take a while."
 COM_JOOMLAUPDATE_UPDATING_FAIL="The update has failed."
 COM_JOOMLAUPDATE_UPDATING_COMPLETE="The update is finalising."
 COM_JOOMLAUPDATE_VIEW_COMPLETE_HEADING="Joomla Version Update Status"
+COM_JOOMLAUPDATE_VIEW_COMPLETE_INSTALLER_MESSAGE="Messages captured by installer"
 COM_JOOMLAUPDATE_VIEW_COMPLETE_MESSAGE="Your site has been updated. Your Joomla version is now %s."
+COM_JOOMLAUPDATE_VIEW_COMPLETE_UPDATE_ERRORS="Brief list of captured errors"
+COM_JOOMLAUPDATE_VIEW_COMPLETE_WITH_ERROR_MESSAGE="The update completed with errors. Please examine the update logs %s for more detail.<br>It is recommended to restore the site from backup, fix the issues that caused the update failure and try to update again.<br>You always can ask for help on Joomla! forum %s"
```

```suggestion
COM_JOOMLAUPDATE_VIEW_COMPLETE_WITH_ERROR_MESSAGE="The update completed with errors. Please examine the update logs %s for more details.<br>It is recommended to restore the site from backup, fix the issues that caused the update failure and try to update again.<br>You always can ask for help on Joomla! forum %s"
```
Same as change suggested by @brianteeman in the other PR #41690 for 5.0-dev.